packages/graphql/src/classes/Neo4jGraphQL.ts

```diff
@@ -108,6 +111,8 @@ class Neo4jGraphQL {
 
             this.authorization = new Neo4jGraphQLAuthorization(authorizationSettings);
         }
+
+        this.complexityEstimatorHelper = new ComplexityEstimatorHelper(!!this.features.complexityEstimators);
```

Comment:
I would personally set the complexityEstimatorHelper as optional and define it only when `features.complexityEstimators` is true, same with the `make-augmented-schema`, I would make the complexityEstimatorHelper dependency optional!